src/core/math.d

```diff
@@ -164,3 +164,101 @@ unittest
     }
 }
 
+/*************************************
+ * Round argument to float precision.
+ *
+ * D language types specify a minimum precision, not
+ * a maximum. toFloatPrec() forces float precision.
+ * Params:
+ *      f = value to convert
+ * Returns:
+ *      f in float precision
+ */
+
+@safe pure nothrow
+float toFloatPrec(float f)  { pragma(inline, false); return f; }
+
+/// ditto
+@safe pure nothrow
+float toFloatPrec(double f) { pragma(inline, false); return cast(float)f; }
+
+/// ditto
+@safe pure nothrow
+float toFloatPrec(real f)   { pragma(inline, false); return cast(float)f; }
+
+@safe unittest
+{
+    static float f = 1.1f;
+    static double d = 1.1;
+    static real r = 1.1L;
```

Comment:
You might want to add a comment to that effect so that no enterprising individual changes it later - especially if there's a decent chance that the tests will pass either way.